test/wallet.js

```diff
@@ -26,8 +26,36 @@ describe('wallet', function () {
     lib.after(mockery)
   })
 
+  afterEach(function () {
+    lib.afterEach(mockery)
+  })
+
   it('create', function () {
     const result = lib.createWallet()
-    snapshot(this.test.fullTitle(), result)
+    snapshot(this.test.fullTitle(), lib.getInfo(result))
+  })
+
+  it('delete', function () {
+    const result = lib.deleteWallet()
+    snapshot(this.test.fullTitle(), lib.getInfo(result))
+  })
+
+  describe('recovery', function () {
+    it('succeeds with a valid key', function () {
+      const result = lib.recoverWallet('wasp broken strong analyst until tray olympic arrow input bicycle gun settle prepare tissue road try sustain husband width brave section obey country area')
+      snapshot(this.test.fullTitle(), lib.getInfo(result))
+    })
+    it('fails with an invalid key', function () {
+      const result = lib.recoverWallet('broken strong analyst until tray olympic arrow input bicycle gun settle prepare tissue road try sustain husband width brave section obey country area')
+      snapshot(this.test.fullTitle(), lib.getInfo(result))
+    })
+    it('recovery does not process with a non-string key', function () {
```

Comment:
can you remove `recovery`? so that we don't have `wallet recovery recovery` in the spec

Reply:
++